---
layout: post
title:  "Fast and highly accurate HLA typing by linearly-seeded graph alignment"
date:   2017-03-22
description: 
categories:
- blog
permalink: HLAPRGLA

---

If you’re interested in immunogenetics, you probably know that high-accuracy HLA type inference from all types of whole-genome sequencing data is now possible: we recently published a tool, [HLA\*PRG](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005151), that enables that.

You might also have noticed, however, that HLA*PRG can be quite resource-hungry – hundreds of CPU hours and up to 80GB of RAM for 2 x 250bp whole-genome sequencing data. Also, it only supported GRCh37.

In this post I describe an algorithm, [HLA\*PRG:LA](https://github.com/AlexanderDilthey/HLA-PRG-LA), that brings down these requirements to ~30GB and ~30 CPU hours per sample, without sacrificing accuracy.

[Continue reading](https://genomeinformatics.github.io/HLA-PRG-LA/) on the Genome Informatics blog...
